styles/components/_buttons.scss

```diff
@@ -109,6 +114,17 @@
   padding: 0 12px;
 }
 
+.iconBtnSm {
+  //this button has only an icon in it
+  @extend .btn;
+  font-size: $tx5;
+  padding: 0;
+  margin: 0;
+  width: 30px;
+  height: 30px;
+  transition: all 0.15s ease;
```

Comment:
Should the transition be on the .btn class, or does only the iconBtnSm class have a transition?

What is supposed to transition? The only thing I saw was the border on hover, if that's the intent it would be better to transition only the border color, so other properties aren't affected by surprise (for instance, if a group of elements was supposed to immediately go to opacity 0, this button would fad slower than the rest of them).